frameworks/hdfs/docs/security.md

```diff
@@ -0,0 +1,182 @@
+example/name-1-zkfc.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+HTTP/name-1-zkfc.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+
+example/journal-0-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+HTTP/journal-0-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/journal-1-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+HTTP/journal-1-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/journal-2-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+HTTP/journal-2-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+
+example/data-0-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+HTTP/data-0-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/data-1-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+HTTP/data-1-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/data-2-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+HTTP/data-2-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+```
+
+{{ include service/security-kerberos-ad }}
+
+{{ include services/security-service-keytab
+    techName="Apache HDFS" }}
+
+#### Install the Service
+
+Install the DC/OS Apache HDFS service with the following options in addition to your own:
+```json
+{
+    "service": {
+        "kerberos": {
+            "enabled": true,
+            "kdc": {
+                "hostname": "<kdc host>",
+                "port": <kdc port>
+            },
+            "primary": "<service primary default hdfs>",
+            "realm": "<realm>",
+            "keytab_secret": "<path to keytab secret>",
+            "debug": <true|false default false>
+        }
+    }
+}
+```
+
+<!-- TO BE DETERMINED *Note*: It is possible to enable Kerberos after initial installation but the service may be unavailable during the transition. Additionally, your HDFS clients will need to be reconfigured. -->
+
+## Authorization
+
+The DC/OS Apache HDFS service supports HDFS's native authorization, which behaves similarly to UNIX file permissions. If Keberos is enabled as detailed [above](#kerberos-authentication), then Kerberos principals are mapped to OS users which HDFS uses to assign permissions internally.
+
+### Enable Authorization
+
+#### Prerequisites
+- Completion of  [Kerberos authentication](#kerberos-authentication) above.
+
+#### Set Kerberos Principal to User Mapping
+
+A custom mapping can be set to map Kerberos principals to OS user names. This is supplied by setting the parameter
+```
+{
+    "hdfs": {
+        "security_auth_to_local": "<custom mapping>"
+    }
+}
+```
+where `<custom mapping>` is a base64 encoded string. The mapping is base64 encoded to ensure it is sent to the service correctly.
+
+**WARNING**: The default mapping is to map **ANY** Kerberos Principal to the OS user `nobody`. This is done for simplicity of testing and you **MUST** change it before placing sensitive data in the system.
```

Comment:
Let's update this to say the default mapping now doesn't exist because of #2275 and that the operator needs to explicitly set the rules.